<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogPrincipal</class>
 <widget class="QDialog" name="dialogPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconocimiento de Objetos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 255, 246);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gráficos por Computador</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Proyecto Interciclo: Reconocimiento de objetos </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Marzo - Agosto 2020 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Integrantes: Jorge Encalada y Vanessa Romero</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSiguiente">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Siguiente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVideo">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRuta">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ruta//</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbOpciones">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Usar la cámara del ordenador</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Subir un video desde el ordenador</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblWebCam">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/WebCam/webcam.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblVideo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Video/video.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../img/Seleccion.qrc"/>
  <include location="../img/ICON.qrc"/>
 </resources>
 <connections/>
</ui>
